src/arb/acb.jl

```diff
@@ -96,6 +119,14 @@ function convert(::Type{ComplexF64}, x::AcbFieldElem)
   return complex(v, w)
 end
 
+function convert(::Type{ComplexF64}, x::AcbFieldElem)
+  return ComplexF64(x)
+end
```

Comment:
@thofma so we have this `convert` method here because it existed before... We can remove it, but technically this is breaking. Thoughts? (I am fine with breaking ; or leaving it in; as long as we agree)